docs/UserGuide.md

```diff
@@ -58,105 +102,154 @@ management tasks done faster than traditional GUI apps.
 An item can have any number of tags (including 0)
-
 </div>
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 Expiry date of an item is optional.
-
 </div>
 
 Examples:
+* `add n/Chocolate Milk l/kitchen refrigerator q/2` adds 2 Chocolate Milk to the kitchen refrigerator. 
+* `add n/Sunshine Bread l/bedroom q/10 e/2020-01-01 t/favourite` adds 10 Sunshine Bread with expiry date, 2020-01-01, 
+  and "favourite" tag to the bedroom.
 
-* `add n/Chocolate Milk l/Kitchen Refrigerator q/2`
-* `add n/Sunshine Bread l/Bedroom q/10 e/2020-01-01 t/favourite`
-
-### Listing all items : `list`
-
-Shows a list of all items in the inventory.
+### 3.3. Editing an item : `edit`
 
-Format: `list` `list [l/LOCATION]` `list [t/TAG]`
+If you want to edit any details of an existing item in the inventory, you can do so by using this command. You don't 
+have to worry about adding any wrong information as you can edit them any time!
 
-* You can view all items in the inventory by typing 'list' without specifying location/tag.
-* The search is case-insensitive. e.g 'room' will match 'Room'.
-* The order of keywords for location search does not matter. e.g. 'Room Living' will match 'Living Room'.
-* Tag keyword only can contain one single word.
-* Only full words will be matched e.g. 'Room' will not match 'Bedroom'.
-* Only location/tag can be searched.
-* Location matching uses each word in the String to do 'AND' search e.g. 'Room' will match 'Living room' but 'Living
-  room 1' will not match 'Living room'
-
-### Editing an item's details : `edit`
+Format: `edit INDEX [n/ITEM_NAME] [l/LOCATION] [q/QUANTITY] [e/EXPIRY_DATE] [t/TAG]…​`
 
-Editing an existing item in the inventory.
+<div markdown="block" class="alert alert-info">
 
-Format: `edit INDEX [n/ITEM_NAME] [l/LOCATION] [q/QUANTITY] [e/EXPIRY_DATE] [t/TAG]…​`
+**:information_source: Notes about the edit command:**<br>
 
 * Edits the item at the specified `INDEX`. The index refers to the index number shown in the displayed item list. The
   index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values **only if input values differ from the existing values**.
 * When editing tags, the existing tags of the item will be removed i.e. adding of tags is not cumulative.
 * You can remove all the items’ tags by typing `t/` without specifying any tags after it.
+</div>
 
 Examples:
-
-* `edit 1 q/10 e/2020-10-11` Edits the quantity of the 1st item to be `10` and expiry date of the 1st item to
-  be `2020-10-11`
-  if the existing quantity and expiry date of the 1st item are both not `10` and `2020-10-11` respectively.
-* `edit 2 n/Chocolate Bread t/` Edits the name of the 2nd item to be `Chocolate Bread` and clears all existing tags if
+* `edit 1 q/10 e/2020-10-11` edits the quantity of the 1st item to be `10` and expiry date of the 1st item to
```

Comment:
same issue with the previous examples, which is likely to be the same for the rest of the examples. Please look into this! Thanks! :)